packages/link/src/__tests__/Link.spec.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+import React from 'react';
+import { shallow } from 'enzyme';
+import Link from '../Link';
+import examples from '../__demo__/examples';
+import testDemoExamples from '../../../../utils/test/testDemoExamples';
+
+function renderLink(props, children) {
+  return shallow(<Link {...props}>{children}</Link>);
+}
+
+describe('Link', () => {
+  let link;
+  it('renders', () => {
+    link = renderLink({ href: 'http://example.com' }, 'Children');
+
+    expect(link.exists()).toEqual(true);
+  });
+
+  describe('onClick', () => {
```

Comment:
This is just default browser behavior. Since we're not doing anything with the `onClick` prop in the component (it just gets passed through, because it's a standard DOM attribute for `a`), we probably don't need tests for it.